custom/v3/commands/data/src/main/java/org/apache/fineract/v3/commands/data/CommandTypedWrapper.java

```diff
@@ -62,9 +49,9 @@ public CommandWrapper getCommandWrapper() {
             throw new RuntimeException(e);
         }
 
-        return new CommandWrapper(officeId, groupId, clientId, loanId, savingsId, actionName, entityName, entityId,
-                subentityId, href, json, transactionId, productId, null, creditBureauId, organisationCreditBureauId,
-                jobName, idempotencyKey);
+        return new CommandWrapper(null, null, null, null, null, actionName, entityName, entityId,
```

Comment:
It's just temporary. Not implementee there to remain.
Actually, I'm breaking some rules to get things to work first. 
Then it can be improved or even removed.